### PR TITLE
Add language use guidelines for contributions made

### PR DESCRIPTION
**Pull request by ShreyaPrabhu**
Summary - Add language use guidelines for contributions made towards adding new term, its definition and examples provided.

Please let me know if anything else is required to be added in the guidelines section. 

Changes to close issue #22

### CODE REVIEW

**Comment by ShreyaPrabhu** [comment]
@ruf-io This PR is ready for review.

File: README.md [hunk 1]
```diff
@@ -84,4 +84,15 @@ The site is statically generated using the [hugo](https://gohugo.io/) framework.
...
\ No newline at end of file
+You're off to the races! Reference [hugo docs](https://gohugo.io/about/) to learn more and start making code contributions.
+
+## Language use guidelines for contributions
+
+All contributions made towards adding a new term and its definition must adhere to these common guidelines:
+
+1. Every contribution made should adhere to the [Code of Conduct](https://github.com/do-community/developer-glossary/blob/main/code_of_conduct.md)
+2. Avoid usage of country specific language when adding a new term, in the term definition and for the examples provided.
+3. Use simple language. Avoid idioms and be clear.
```
Comment (ruf-io) [inline README.md]: Can you  link "Use simple language" to here: https://plainlanguage.gov/resources/articles/dash-writing-tips/

**Review by ruf-io — COMMENTED** [review]
One link to add, then we're good! Thanks @ShreyaPrabhu

File: README.md (hunk 1 above)
Comment (ShreyaPrabhu) [reply README.md]: Yeah sure.

Comment (ShreyaPrabhu) [reply README.md]: @ruf-io I have made the change

**Pull request merged**